aiohttp/pytest_plugin.py

```diff
@@ -129,6 +129,13 @@ def _runtime_warning_context():  # type: ignore
                 '\n'.join(rw)
             ))
 
+    # Propagate warnings to pytest
+    for msg in _warnings:
+        warnings.showwarning(
+            msg.message,   # type: ignore
```

Comment:
Python supports passing `Warning` instance as well as string, mypy is not aware. We can fetch message from the `Warning` instance but this way some extra information stored within `Warning` may be lost.